appinfo/info.xml

```diff
@@ -2,16 +2,33 @@
 <info>
     <id>openidconnect</id>
     <name>OpenID Connect</name>
-    <summary>OpenID Connect Integration</summary>
-    <description>OpenID Connect Integration with ownCloud</description>
+    <summary>User authentication and SSO with OpenID Connect (OIDC)</summary>
+    <description>
+This extension adds support for [OpenID Connect](https://openid.net/connect/) to ownCloud Server. 
+
+OpenID Connect has become the leading standard for single sign-on and identity provision on the Internet. With ownCloud it can be used for user authentication and client authorization against an external Identity Provider.
+
+**Benefits of using ownCloud with OpenID Connect**
+
+- Increased security by shifting user authentication to an external Identity Provider
+- Seamless integration into single sign-on (SSO) environments as well as with third party products
+- Centralized client management within the Identity Provider
+- Enterprise-grade security through the use of authentication security features (e.g., multi-factor authentication) and policies (e.g., automatic token expiration on certain conditions) provided by Identity Providers
+
+The integration requires an Identity Provider which supports OpenID Connect (e.g., Keycloak, Kopano Konnect, Ping Federate, ADFS or others).
+More information on setup, configuration and migration can be found in the ownCloud Documentation.</description>
     <licence>GPLv2</licence>
     <author>Thomas Müller</author>
     <version>0.1.0</version>
```

Comment:
Version should be 0.2.0 according https://github.com/owncloud/openidconnect/pull/65

Reply:
please rebase